frontend/src/utils/share.js

```diff
@@ -0,0 +1,10 @@
+export const generateShareItemLabel = (itemType): string => {
+  switch (itemType) {
+    case 'Table':
+      return 'Table';
+    case 'S3Bucket':
+      return 'S3Bucket';
+    case 'StorageLocation':
+      return 'Folder';
+  }
+};
```

Comment:
I am not sure this is the best place to place this function. It is only used by Shares/components and /views. I lean towards placing code closer to its usage, and if it is shared across multiple modules, then add it to the correspondent functional directory. Let me review with the frontend designers, because what usually applies for backend does not always apply to frontend.

Reply:
Sounds good! Please let me know what the conclusion is and I can make the changes.

Reply:
I would move it to `frontend/modules/Shares`